x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/views/meta_engine_schema.tsx

```diff
@@ -40,7 +42,75 @@ export const MetaEngineSchema: React.FC = () => {
         )}
       />
       <FlashMessages />
-      <EuiPageContentBody>TODO</EuiPageContentBody>
+      <EuiPageContentBody>
+        {hasConflicts && (
+          <>
+            <EuiCallOut
+              iconType="alert"
+              color="warning"
+              title={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.schema.metaEngine.conflictsCalloutTitle',
+                {
+                  defaultMessage:
+                    '{conflictingFieldsCount, plural, one {# field is} other {# fields are}} not searchable',
```

Comment:
I'm not actually sure this i18n is correct. Does anyone know for sure how to handle `is` vs. `are` pluralization in i18n?... 😬